### PR TITLE
[LT-1115] Accept multiple events

### PR DESCRIPTION
**Pull request by jc-sailthru**

### CODE REVIEW

File: src/main/java/com/sailthru/sqs/MParticleClient.java [hunk 1]
```diff
@@ -51,22 +52,24 @@ private EventsApi getEventsApi(final String apiKey, final String apiSecret) {
         return apiClient.createService(EventsApi.class);
     }
 
-    private Batch prepareBatch(final MParticleMessage message) {
+    private Batch prepareBatch(final MParticleOutgoingMessage message) {
         final Batch batch = new Batch();
         batch.environment(Batch.Environment.DEVELOPMENT);
         batch.userIdentities(new UserIdentities()
                 .email(message.getProfileEmail())
         );
```
Comment (bge-kernel-panic) [inline src/main/java/com/sailthru/sqs/MParticleClient.java:61]: todo: add the timestamp here:

```java
batch.timestampUnixtimeMs(parseTimestamp(message.getTimestamp()));
// ...

private Long parseTimestamp(String timestamp) {
    try {
        if (!Strings.isEmpty(timestamp)) {
            final ZonedDateTime dt = ZonedDateTime.parse(timestamp, ISO_DATE_TIME);
            return dt.toInstant().toEpochMillis();
        }
    } catch (DateTimeParseException e) {
        // maybe log the error
    }
    return null;
}
```

**Review by bge-kernel-panic — CHANGES_REQUESTED** [review]
I believe it's pretty important to make use of the timestamp lest trouble occur when we process a backlog of messages.

File: src/main/java/com/sailthru/sqs/MParticleClient.java (hunk 1 above)
Comment (jc-sailthru) [reply src/main/java/com/sailthru/sqs/MParticleClient.java:68]: Good catch, this is done

**Review by bge-kernel-panic — APPROVED** [review]
+1

**Pull request merged**